shlr/sdb/src/disk.c

```diff
@@ -12,49 +12,34 @@
 #if __SDB_WINDOWS__
```

Comment:
please,  if you do changes. in the sdb directory , make first a separate PR in the sdb repository

Reply:
should I remove these changes from this PR?

Reply:
done https://github.com/radare/sdb/pull/170